pkg/server/namespace.go

```diff
@@ -3,16 +3,19 @@ package server
 import (
 	"encoding/json"
 	"net/http"
+
+	"github.com/openfaas/faas-netes/pkg/handlers"
+	"k8s.io/client-go/kubernetes"
 )
 
-func makeListNamespaceHandler(defaultNamespace string) func(http.ResponseWriter, *http.Request) {
+func makeListNamespaceHandler(defaultNamespace string, clientset kubernetes.Interface) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		res := handlers.ListNamespaces(defaultNamespace, clientset)
```

Comment:
Where did `		defer r.Body.Close()		` go ?

Reply:
Well this function is literally taken from `handlers.MakeNamespacesLister` which does not have `defer r.Body.Close()`.

On another note I could have just reuse `handlers.MakeNamespacesLister` instead of copy pasting, but this followed how the other handlers are implemented for the operator logic and bigger refactoring should take place as per Lucas's comment from the previous review